www/js/lib/popovers.js

```diff
@@ -104,14 +104,12 @@ function cleanUpLedeContent (node) {
     styleElements.forEach(style => {
         style.parentNode.removeChild(style);
     });
-    const paragraphs = Array.from(node.querySelectorAll('p'));
-    // Filter out empty paragraphs or those with less than 50 characters
+    const paragraphs = Array.from(node.querySelectorAll('p:not(#pcs-edit-section-title-description)'));
     const parasWithContent = paragraphs.filter(para => {
         // DEV: Note that innerText is not supported in Firefox OS, so we need to use textContent as a fallback
         // The reason we prefer innerText is that it strips out hidden text and unnecessary whitespace, which is not the case with textContent
         const innerText = para.innerText ? para.innerText : para.textContent;
-        const text = innerText.trim();
-        return !/^\s*$/.test(text) && text.length >= 50;
```

Comment:
I don't understand why you've deleted these lines and replaced it with the one below. This reduces cleanup functionality. Please restore.

Reply:
Ohhhhh sorry sorry!!!!! I was playing with the code to see what will happen to website and pushed it without fixing it. fixing this right away.